lib/tokens.js

```diff
@@ -18,9 +18,9 @@ import pLimit from 'p-limit';
 import {promisify} from 'util';
 import Bitstring from '@digitalbazaar/bitstring';
 import {IdGenerator} from 'bnid';
-import {constants as citConstants, appContextMap, documentLoader}
+import {constants as citConstants, documentLoader}
   from 'cit-context';
-const {CIT_CONTEXT_URL} = citConstants;
+const {CONTEXT_URL} = citConstants;
```

Comment:
I think it's better to be explicit here about which context it is.

```suggestion
const {CONTEXT_URL: CIT_CONTEXT_URL} = citConstants;
```